src/components/message-input/menu/menu.tsx

```diff
@@ -1,38 +1,78 @@
 import React from 'react';
 import Dropzone from 'react-dropzone';
-import { dropzoneToMedia, Media } from '../utils';
+import { bytesToMB, dropzoneToMedia, Media } from '../utils';
 import { IconPaperclip } from '@zero-tech/zui/icons';
 import { IconButton } from '../../icon-button';
 
 import './styles.scss';
+import { ToastNotification } from '@zero-tech/zui/components';
+import { config } from '../../../config';
 
 export interface Properties {
   onSelected: (images: Media[]) => void;
   mimeTypes?: any;
   maxSize?: number;
 }
 
-export default class Menu extends React.Component<Properties> {
+interface State {
+  isDropRejectedNotificationOpen: boolean;
+}
+
+export default class Menu extends React.Component<Properties, State> {
+  state = { isDropRejectedNotificationOpen: false };
+
   imagesSelected = (acceptedFiles): void => {
+    this.setState({ isDropRejectedNotificationOpen: false });
+
     const newImages: Media[] = dropzoneToMedia(acceptedFiles);
 
     if (newImages.length) {
       this.props.onSelected(newImages);
     }
   };
 
+  renderToastNotification = () => {
+    const maxSize = bytesToMB(config.cloudinary.max_file_size);
+
+    return (
+      <ToastNotification
+        title={''}
+        description={`File exceeds ${maxSize} size limit`}
+        actionTitle=''
```

Comment:
I think with your changes to the component in zUI you can remove the `actionTitle` attribute ?